src/test/java/org/sigmah/server/i18n/I18nServerTest.java

```diff
@@ -22,53 +22,50 @@
  * #L%
  */
 
-
-import org.junit.Assert;
-import org.junit.Ignore;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.sigmah.server.inject.ConfigurationModule;
-import org.sigmah.server.inject.GuiceJUnitRunner;
-import org.sigmah.server.inject.GuiceJUnitRunner.GuiceModules;
-import org.sigmah.server.inject.I18nServerModule;
-import org.sigmah.server.inject.PersistenceModule;
-import org.sigmah.shared.Language;
-
-import com.google.inject.Inject;
-import java.io.IOException;
-import java.util.Properties;
-
-/**
- * JUnit test for the {@link I18nServer} Module
- * 
- * @author Maxime Lombard (mlombard@ideia.fr)
- */
-@RunWith(GuiceJUnitRunner.class)
-@GuiceModules({
-								PersistenceModule.class,
-								ConfigurationModule.class,
-								I18nServerModule.class
-})
-// TODO use Assume to skip test if environment is not set
-@Ignore("Test requires a database environment")
-public class I18nServerTest {

```

Comment:
This file had mixed line ending. The first lines were \n whereas those marked as modified were \r\n. I made the changes with `vi` to preserve those but it seems that the commit lost them.